pb_model/fields.py

```diff
@@ -195,6 +195,9 @@ def from_pb(instance, dj_field_name, pb_field, pb_value):
         setattr(instance, dj_field_name, [related_model().from_pb(pb_message) for pb_message in pb_value])
 
 
+class RepeatedForeignField(JSONField):
```

Comment:
What's this class for?
Should we add docstring and usage instruction in readme?